src/extension/csw/core/src/test/java/org/geoserver/csw/GetRecordsTest.java

```diff
@@ -590,6 +591,7 @@ public void testValidatePost() throws Exception {
     }
 
     @Test
+    @Ignore("We no longer allow empty escapes")
```

Comment:
This was introduced by GEOS-7550, see the reason why we did not go for throwing an error here:
https://osgeo-org.atlassian.net/browse/GEOS-7550